code401challenges/Data-Structures/src/test/java/datastructure/LibraryTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package datastructure;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LibraryTest {
+    Library.LinkedList list = new Library.LinkedList();
+
+    @Before
+    public void initializeForTest() {
+        list.insert(1);
+        list.insert(4);
+        list.insert(7);
+        list.insert(13);
+        list.insert(17);
+        list.insert(19);
+        list.insert(23);
+        list.insert(29);
+        list.insert(37);
+        list.insert(59);
+        list.insert(60);
+    }
+
+
+
+    //Check to see if empty linked list gets created
+    @Test
+    public void emptyList() {
+        Library.LinkedList emptyList = new Library.LinkedList();
+        assertEquals("HEAD -> NULL", emptyList.toString());
+    }
+
```

Comment:
some messages in your tests would be helpful, or comments before hand that explain what the test is meant to achieve. at the very least, numerically naming them is not helpful